### PR TITLE
Add more extraction code

### PR DESCRIPTION
**Pull request by eyurtsev**
Add more extraction code

### CODE REVIEW

File: langchain_benchmarks/extraction/implementations.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from langchain.schema.runnable import Runnable
+from langsmith.client import Client
+from pydantic import BaseModel
+
+from langchain_benchmarks.extraction.evaluators import get_eval_config
+from langchain_benchmarks.schema import ExtractionTask
+
+# PUBLIC API
+
+def create_openai_function_based_extractor(
```
Comment (hinthornw) [inline langchain_benchmarks/extraction/implementations.py:16]: ooc: why are these diff than LangChain?

Comment (eyurtsev) [reply langchain_benchmarks/extraction/implementations.py:16]: what are they in langchain? I just cargo culted some code?

Comment (hinthornw) [reply langchain_benchmarks/extraction/implementations.py:17]: well there was create_extraction_chain but i guess we no longer use it

**Review by hinthornw — APPROVED** [review]

File: langchain_benchmarks/extraction/implementations.py (hunk 1 above)
Comment (eyurtsev) [reply langchain_benchmarks/extraction/implementations.py:17]: will review right now if there's a way to re-use it

**Pull request merged**